src/components/Popup.jsx

```diff
@@ -100,6 +100,7 @@ class Popup extends React.Component {
                       enabled={(btn.enabled !== undefined) ? btn.enabled : true}
                       buttonType='default'
                       value={btn.value}
+                      aria-labelledby='popup__message'
```

Comment:
@tianj7 could you please touch base with @cemile-ctds regarding the business requirements? Usually the aria-labelledby acts by referencing the id of another element that provides the label text for screen readers. I believe if we have aria-labelledby we would not need the aria-labels you have added. Also make sure the element with the id `popup_message` is the correct element to act as the label.